client/pwa/src/db.ts

```diff
@@ -1,3 +1,8 @@
+// WARNING - This file is duplicated at two locations:
```

Comment:
@caugner @Guyzeroth I'd try to not expose the whole DB to the client. Maybe we need to read up but I'm worried about connecting to two different versions to the DB both with write access. Take a look at: https://gist.github.com/fiji-flo/22933db64b2e4772b8c9f122e1b1aada

That works doesn't expose everything to the front-end.

Reply:
I hadn't seen a way to connect read-only from the client, but your approach looks reasonable. Will try it out.